packages/cojson/src/coValues/coList.ts

```diff
@@ -298,8 +323,13 @@ export class RawCoList<
    *
    * @category 4. Time travel
    */
-  atTime(_time: number): this {
-    throw new Error("Not yet implemented");
+  atTime(time: number): this {
+    if (time >= (this.lastValidTransaction ?? 0)) {
+      return this;
```

Comment:
This could potentially lead to bugs, because this doesn't have a time filter and new transactions might "overflow".

We needed this check in CoMap because time traviling there is used to rollback Group in time and use it to validate permissions rights on transactions.

This case is not that performance-critical, so it's ok to go always for the slow path (for now)